docs/charter.md

```diff
@@ -36,9 +76,7 @@ ICU4X will have an independent code base from ICU, and will operate independentl
 
 ### Is ICU4X going to replace ICU?
 
-No!
```

Comment:
Why did you delete this line? Please add it back unless you think people need to read the full paragraph in order to answer this question.

Reply:
Because it doesn't add a lot of value and doesn't feel like an official document.
A paragraph is not long so I think a more neuanced answer should be ok. 

I can bring it back, if you are strongly in favor of keeping it. Just sync with @nciric on how it should look like.

Reply:
OK, if @zbraniecki and @nciric think the line is not necessary, let's remove it.  I just wanted to make sure it wasn't removed accidentally.